tcms/telemetry/static/telemetry/js/testing/breakdown.js

```diff
@@ -18,13 +18,13 @@ $(document).ready(() => {
 function reloadCharts() {
```

Comment:
While reloading charts & the test plan dropdown field works  with multiple select the Product widget stays open after I click it. The onclick handler should explicitly close the widget otherwise it obscures parts of the charts and also other widgets.

Reply:
On a page with about 20 products the drop down for Product obscures half of the chart and I know that selecting multiple products reloads the chart but that isn't visible at all. 

IMO the charts are the more important thing and if we have to choose between which one to be visible I will give preference to the charts instead of the select widgets.

Reply:
Yeah that makes sense. I'll make this change